pkg/kubelet/cri/streaming/server.go

```diff
@@ -146,9 +146,10 @@ func NewServer(config Config, runtime Runtime) (Server, error) {
 	handler.Add(ws)
 	s.handler = handler
 	s.server = &http.Server{
-		Addr:      s.config.Addr,
-		Handler:   s.handler,
-		TLSConfig: s.config.TLSConfig,
+		Addr:              s.config.Addr,
+		Handler:           s.handler,
+		TLSConfig:         s.config.TLSConfig,
+		ReadHeaderTimeout: 30 * time.Minute,
```

Comment:
should this be configurable.. or at least pulled from a const specified in a common location for such timeout values?